### PR TITLE
Avoid NPE from CaseResult.getName()

### PR DESCRIPTION
**Pull request by jglick**
See https://github.com/jenkinsci/junit-plugin/pull/135#issuecomment-686469488. CC @mguillem

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by jglick** [comment]
Yes I would like to do that. Looks like I have release perms. Thanks!

**Comment by jglick** [comment]
https://github.com/jenkinsci/junit-plugin/releases/tag/junit-1.33

File: src/test/java/hudson/tasks/junit/CaseResultTest.java [hunk 1]
```diff
@@ -271,6 +271,14 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         return rule.assertBuildStatus(Result.UNSTABLE, p.scheduleBuild2(0).get());
     }
 
+    @Test
+    public void emptyName() throws Exception {
```
Comment (mguillem) [inline src/test/java/hudson/tasks/junit/CaseResultTest.java:275]: `missingName` instead of `emptyName`?

Comment (jglick) [reply src/test/java/hudson/tasks/junit/CaseResultTest.java:275]: Yeah I suppose so.

**Pull request merged**